ci/prow/make_config.go

```diff
@@ -1371,9 +1176,16 @@ func main() {
 			generateIssueTrackerPeriodicJobs()
 			generatePerfClusterUpdatePeriodicJobs()
 		}
-		generateOtherJobConfigs("postsubmits", func(repo repositoryData) bool {
-			return repo.EnableGoCoverage
-		}, generateGoCoveragePostsubmit)
+		for _, repo := range repositories {
+			if repo.EnableGoCoverage {
+				generateGoCoveragePostsubmit("postsubmits", repo.Name, nil)
+				// generate post submit jobs to reconcile clusters for perf
+				// testing of the given repo. we need to run it in the "if" block, this is
+				// a bit hacky as we have to group the postsubmit jobs for each repo.
+				// TODO(chizhg): use a more generic way to generate the config.
+				generatePerfClusterReconcilePostsubmitJob(repo.Name)
```

Comment:
This is buggy, as these postsubmits are unrelated. But you already know that. However, your comment doesn't explain why it's so hard to generate them the proper way.

Reply:
Changed to use setting in `config_knative.yaml`